### DISCUSSION TITLE
Redis 7 commands to implement

### DISCUSSION THREAD

**Discussion by AvitalFineRedis** [xetorthio/jedis#2807] [General]
- [ ] SPUBLISH, SSUBSCRIBE, SUNSUBSCRIBE, PUBSUB SHARDCHANNELS/SHARDNUMSUB (redis/redis#8621)
- [ ] CLUSTER SLOTS hostname support (redis/redis#9530)
- [x] ACL DRYRUN (redis/redis#9974)
- [x] XADD supports auto sequence number via -* (redis/redis#9217)
- [x] CONFIG SET/GET can handle multiple configs atomically, in one call (redis/redis#9748, redis/redis#9914)
- [x]  LCS
- [x]  LMPOP #2791 
- [x]  BLMPOP #2789 
- [x]  ZMPOP #2794 
- [x]  BZMPOP #2793 
- [x]  EXPIRETIME
- [x]  PEXPIRETIME
- [x]  PEXPIRE - new arguments [NX|XX|GT|LT]
- [x]  SINTERCARD
- [x]  ZINTERCARD
- [x]  SORT_RO
- [x]  EVAL RO
- [x]  EVALSHA_RO
- [x]  CLUSTER ADDSLOTSRANGE #2792 
- [x]  CLUSTER DELSLOTSRANGE #2795 
- [x]  CLIENT NO-EVICT
- [x] SHUTDOWN - [FORCE] [ABORT] arguments

**Comment by Avital-Fine** [comment]
@sazzad16 Can you please add [COMMAND GETKEYSANDFLAGS](https://redis.io/commands/command-getkeysandflags)

**Reply by sazzad16** [reply]
@Avital-Fine Not necessary. 

Jedis has none of `COMMAND ...`. If we want to add just one variant, we should be adding all of them.
